test/suites/osconfig/clusterid.robot

```diff
@@ -83,10 +102,79 @@ Get MicroShift Cluster ID From Namespace
     Should Not Be Empty    ${clusterid}
     RETURN    ${clusterid}
 
+Extract Sos Report
+    [Documentation]    Extract Sos Report from the tar file
+    [Arguments]    ${sos_report_tarfile}
+
+    ${sos_report_dir}    ${rc}=    Execute Command
+    ...    dirname ${sos_report_tarfile}
+    ...    sudo=True    return_rc=True    return_stdout=True
+    Log    ${sos_report_dir}
+    Should Be Equal As Integers    0    ${rc}
```

Comment:
Please, use RF-native functionality for path parsing, i.e. `Operating System.Get Parent Directory`

Reply:
Sorry, we should use `Split Path` keyword from https://robotframework.org/robotframework/latest/libraries/OperatingSystem.html#Split%20Path

Reply:
thanks, added `Split Path` keyword